src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -289,4 +242,73 @@ public String availability(Model model) {
         }
         return allGameNames;
     }
+
+    @GetMapping("/room")
+    @ResponseBody
+    public List<Room> fetchAllRooms() {
+        return roomService.fetchAll();
+    }
+
+    @GetMapping("/room/{id}/")
+    public ResponseEntity fetchRoomById(@PathVariable("id") int id) {
+        Room foundRoom = roomService.fetchById(id);
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        return new ResponseEntity(foundRoom, headers, HttpStatus.OK);
+    }
+
+    @PostMapping(value = "/Game", consumes = "application/json", produces = "application/json")
+    public Game createGame(@RequestBody Game game) {
+        return game;
+
+    }
+
+    @PostMapping(value = "/ProfilePicture", consumes = "application/json", produces = "application/json")
+    public ProfilePicture createProfilePicture(@RequestBody com.TableTalk.Enterprise.dto.ProfilePicture profilePicture) {
+        return profilePicture;
+    }
+
+
+    @PostMapping(value = "/User", consumes = "application/json", produces = "application/json")
+    public User createUser(@RequestBody com.TableTalk.Enterprise.dto.User user) {
+        return user;
+    }
+
+    @PostMapping(value = "/room", consumes = "application/json", produces = "application/json")
+    public Room createRoom(Room room) throws Exception {
+        Room newRoom = null;
+        roomService.save(room);
+        return room;
+    }
+
+    @DeleteMapping("/Game")
+    public ResponseEntity deleteGames() {
+        return new ResponseEntity(HttpStatus.OK);
+    }
+
+    @DeleteMapping("/ProfilePicture")
+    public ResponseEntity deleteProfilePicture() {
+        return new ResponseEntity(HttpStatus.OK);
+    }
+
+    @DeleteMapping("/deleteRoom/{id}/")
+    public ResponseEntity deleteRoom(@PathVariable("id") int id) {
+        log.debug("Entering delete room endpoint");
+        try {
+            roomService.delete(id);
+            log.info("Room with ID " + id + " was deleted.");
+            return new ResponseEntity(HttpStatus.OK);
+        } catch (Exception e) {
+            log.error("Unable to delete room with ID: " + id + ", message: " + e.getMessage(), e);
+            return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+
+    }
+
+    @DeleteMapping("/User")
+    public ResponseEntity deleteUser() {
+        return new ResponseEntity(HttpStatus.OK);
+    }
+
+
```

Comment:
an example of re-ordering everything.

Reply:
Does this re-ordering reduce technical debt?

I'm reluctant to re-order much in a commit, as it then makes future comparisons more difficult.  That can also lead to more potential for merge conflicts.  So, if there's a good justification based on technical debt... OK.  But if it's just reordering for the sake of reordering, I'd skip it.